doc/client_lua_api.md

```diff
@@ -1332,8 +1332,11 @@ It can be created via `Raycast(pos1, pos2, objects, liquids)` or
 
 ```lua
 {
-    hud_elem_type = "image", -- see HUD element types, default "text"
+    type = "image", -- see HUD element types, default "text"
 --  ^ type of HUD element, can be either of "image", "text", "statbar", or "inventory"
+--	If undefined "text" will be used.
```

Comment:
This is redundant as it says above - `default "text"`

Reply:
Fixed, interesting how i already existed in the `client_lua_api,md` but not in `lua_api.md`